EPR/Library/tests/test_noise.cpp

```diff
@@ -0,0 +1,73 @@
+#define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
+
+#include <iostream>
+
+#include "doctest.h"
+#include "../noise.hpp"
+
+
+// Function to scale double values to integer
+// used for comparison.
+int scale_double(double num){
+    double scale = 0.0001;
+    return (int)round((int)(num/scale) * scale * 10000);
+}
+
+TEST_CASE("Testing noise: Simple test"){
+    srand(10);
```

Comment:
+1 avoiding rand() and srand()

Reply:
I will need to use srand() since the c version uses rand(). This make sure that random sequence will be the same when running the test for the c version (and also the c++ version). It is hard to test randomness without it.

Reply:
Fixed